components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/BuildScanDataFields.java

```diff
@@ -81,4 +78,8 @@ private static String summaryTotal(BuildScanData data, String avoidanceOutcome,
     private static String formatDuration(Duration duration) {
         return duration == null ? NO_VALUE : format(duration);
     }
+
+    private static String formatSerializationFactor(BigDecimal serializationFactor) {
```

Comment:
Why is the serialization factor a BigDecimal?

Reply:
Jim had the same comment in a previous PR. 

I've basically been conditioned to always default to using `BigDecimal` and to avoid `double` / `float` primitive types for representing floating point numbers [to avoid any chance of losing precision](https://stackoverflow.com/questions/588004/is-floating-point-math-broken).

I am fine to change it in a subsequent PR.